tools/setup/setup_venvs.py

```diff
@@ -15,11 +15,14 @@
 
 PY2_DEV_REQS_FILE = os.path.join(ZULIP_PATH, "requirements", "py2_dev.txt")
 PY3_DEV_REQS_FILE = os.path.join(ZULIP_PATH, "requirements", "py3_dev.txt")
+THUMBOR_REQS_FILE = os.path.join(ZULIP_PATH, "requirements", "thumbor.txt")
 
 PY2 = sys.version_info[0] == 2
 
 def main(is_travis=False):
     # type: (bool) -> None
+    setup_virtualenv("/srv/thumbor-venv", THUMBOR_REQS_FILE, patch_activate_script=True)
```

Comment:
Since we're planning to move to Python 3 only, let's start with Thumbor as Python 3 only.